plugin/Record/RecordManagerUpdateListViewAddress.php

```diff
@@ -94,7 +94,10 @@ private function insertNewRecords(array $records, string $table): bool
 		$pWpDb = $this->getWpdb();
 		$result = true;
 		foreach ($records as $record) {
-			$result = $result && $pWpDb->insert($prefix.$table, $record);
+			if(is_array($record))
```

Comment:
This is better. @vu-vuong-esg do you know, under which circumstances `$record` is not an array?

Reply:
because $record is string so it doesn't know column in table insert.

Reply:
In which case is `$record` a string? What is the root cause?

Reply:
@vu-vuong-esg are you sure `$row` has the correct type here and doesn't need to be wrapped in another array?
https://github.com/onOfficeGmbH/oo-wp-plugin/blob/f328ccafd4dc37a672ab9d33b156d00e071ac263/plugin%2FGui%2FAdminPageAddressListSettings.php#L233-L236